pages/api/polling/precheck.ts

```diff
@@ -29,15 +30,17 @@ export default withApiHandler(async (req: NextApiRequest, res: NextApiResponse)
   const cacheKey = getRecentlyUsedGaslessVotingKey(voter);
 
   // TODO add a check to see if user has already voted in polls?
```

Comment:
Not part of your updates but noticed this is a comment that can be removed now (if you make any additional PR updates)